test/include.test.js

```diff
@@ -1254,6 +1254,24 @@ describe('include', function() {
     });
   });
 
+  it('should not throw on fetch User if include is boolean equals true', function(done) {
+    User.find({include: true}, function(err, users) {
+      should.not.exist(err);
+      should.exist(users);
+      users.length.should.be.ok;
```

Comment:
here we should assert that the length is 0, since it returns an empty array.

Reply:
Hi @jannyHou, the include in `./lib/include.js:51` normalizes to an empty array. `users` are returned as if there would have been no include statement in the find.  
users.length is equal 5 in this testcase.

Reply:
I believe `ok` must be called to actually execute the assertion.

```suggestion
      users.length.should.be.ok();
```

I find it weird to assert that a length (a number) is ok. Can we find a better assertion, e.g. "greater than zero" or `.not.empty()`?

```suggestion
      users.should.not.be.empty();
```

I don't remember Should.js APIs from the top of my head, I hope `empty()` is a valid assertion 🙄